src/locales/en.json

```diff
@@ -535,21 +535,20 @@
       "title": "Report this content",
       "issue": "What's the issue?",
       "issue-description": "Please describe the issue for us",
-      "copyright": "Infringes Copyright",
+      "dmca": "Infringes Copyright",
       "mature": "Contains mature content",
       "other": "Other",
       "caption": "For security purposes, Openverse collects and retains anonymized IP addresses of those who complete and submit this form.",
       "next": "Next",
       "back": "Back",
       "submit": "Submit",
-      "dmca": {
-        "content": "You must fill out this {link} to report copyright infringement. No action will be taken until the form is filled out and submitted.",
-        "provider": "We recommend doing the same at the source, {link}.",
-        "thanks": "Thank you for reporting an issue with the results of Openverse!",
-        "submission-error": "There was an error with the report submission. Please try again.",
-        "link": "DMCA form",
-        "aria-form": "{provider} form"
-      }
+      "thanks": "Thank you for reporting an issue with the results of Openverse!",
+      "provider": "We recommend doing the same at the source, {link}.",
+      "link": "DMCA form",
+      "aria-form": "{provider} form",
+      "submission-error": "There was an error with the report submission. Please try again.",
+      "dmca-content": "You must fill out this {link} to report copyright infringement. No action will be taken until the form is filled out and submitted.",
+      "placeholder": "Issue description required (with at least 20 characters)"
```

Comment:
What is the reason to flatten these and remove them from a 'dmca' namespace? This is a nitpick, just curious

Reply:
It was originally because the 'DoneMessage' also used strings from 'dmca'. But I should have only flattened the strings that are used by both, and kept the dmca-only strings in the nested object.